source/components/panel/index.html

```diff
@@ -0,0 +1,75 @@
+---
+title: Panel
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 class="hxSectionTitle" id="summary">Summary</h2>
+    <p>
+      A panel is a utility for sectioning content vertically into a head,
+      body, and foot.
+    </p>
+    <ul>
+      <li>The head is fixed to the top of the panel.</li>
+      <li>The foot is fixed to the bottom of the panel.</li>
+      <li>The body fills the space in the middle.</li>
+    </ul>
+  </section>
+  <section id="vue-panelDemo">
+    <h2 class="hxSectionTitle" id="demo">Demo</h2>
+    <div class="hxRow">
+      <div class="hxCol hxSpan-12-xs hxSpan-8-md">
+        <div class="demo panel-demo">
+          <hx-panel>
+            <hx-panelhead v-if="hasHead">Panel Head</hx-panelhead>
+            <hx-panelbody>
+              <p>Panel Body</p>
+              <p>
+                Try to decide the branch off team, maybe it will help spike the MVP
+                interfaces. If we practice between the tasks, we can get the technical
+                application like the VOC branching. The best way to document acceptance
+                impediment is to periodically build their meeting. It was discovered
+                that by efficiently releasing the MVP products, we can practice the
+                lean XP development team in the minor board.
+              </p>
+            </hx-panelbody>
+            <hx-panelfoot v-if="hasFoot">Panel Foot</hx-panelfoot>
+          </hx-panel>
+        </div>
+        <small class="hxItalic">
+          *Visual styling added for demonstration purposes.
+        </small>
+      </div>
+      <div class="hxCol hxSpan-12-xs hxSpan-4-md">
+        <hx-panel>
+          <hx-panelhead>
+            <small>
+              Resize the demo to see how the panel reacts.
+            </small>
+          </hx-panelhead>
+          <hx-panelbody class="hxBox-md">
+            <p>
+              <label>
+                <input type="checkbox" v-model="hasHead" />
+                Show Head
+              </label>
+              <br />
+              <label>
+                <input type="checkbox" v-model="hasFoot" />
+                Show Foot
+              </label>
+            </p>
+          </hx-panelbody>
+        </hx-panel>
+      </div>
+    </div>
+    <br />
+    {% code 'html' %}
+      <hx-panel>
+        [<hx-panelhead></hx-panelhead>]
```

Comment:
are the `[]` around the head and footer for optional? will users naturally know that?

Reply:
This is common syntax that many developers have likely seen. It's used in man pages for CLI programs and it's also used in the [EncoreUI ngdocs](http://rackerlabs.github.io/encore-ui/5.x/ngdocs/index.html#/api/elements.directive:rxBulkSelect).

Reply:
If there is confusion, we can address it as it is brought to light.